backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -70,14 +75,18 @@ public record CreateProjectResponse(Guid? Id, CreateProjectResult Result);
         permissionService.AssertCanManageProject(input.ProjectId);
         var project = await dbContext.Projects.FindAsync(input.ProjectId);
         if (project is null) throw new NotFoundException("Project not found");
-        var user = await dbContext.Users.FindByEmail(input.UserEmail);
+        var user = await dbContext.Users.FindByEmailOrUsername(input.UserEmail);
         if (user is null)
         {
             var manager = loggedInContext.User;
             await emailService.SendCreateAccountEmail(input.UserEmail, input.ProjectId, input.Role, manager.Name, project.Name);
             throw new ProjectMemberInvitedByEmail("Invitation email sent");
         }
-        if (!user.EmailVerified) throw new ProjectMembersMustBeVerified("Member must verify email first");
+        if (user.CreatedBy is null && !user.EmailVerified)
```

Comment:
`user.CreatedBy` will be null if `CreatedBy` isn't `Include()`ed right?
```suggestion
        if (user.CreatedById is null && !user.EmailVerified)
```

Reply:
Yep, I just realized that myself. I ended up creating an extension method on User rather than simply change CreatedBy to CraetedById, because I realized that the logic we want is going to depend on the role they're being assigned.

Reply:
That's correct

Reply:
Done in commit e1391d6498280e77fd10d35730508154d351e23b.